source/features/avoid-accidental-submissions.tsx

```diff
@@ -0,0 +1,34 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function onKeyDownEnter(event: delegate.Event<KeyboardEvent, HTMLFormElement>): void {
+	if (event.key === 'Enter' && !event.ctrlKey) {
+		event.preventDefault();
+		select('#issue_body, #pull_request_body, #commit-description-textarea', event.delegateTarget.form)!.focus();
+	}
+}
+
+// Add ability to submit via Ctrl+Enter in isNewFile and isEditingFile pages (was not available before)
+function onKeyDownCtrlEnter(event: delegate.Event<KeyboardEvent, HTMLFormElement>): void {
+	if (event.key === 'Enter' && event.ctrlKey) {
+		event.delegateTarget.form.submit();
+	}
+}
```

Comment:
Why cant we do

https://github.com/sindresorhus/refined-github/blob/8ca75a97bf92ade57b220f9945a06e3eee865c9c/source/features/create-release-shortcut.tsx#L6-L8

Reply:
Added in https://github.com/sindresorhus/refined-github/pull/4252/commits/e3645e92814c6aa2874e8871f09faebd7239708c

Reply:
Can  you add the shortcut (look at that file to see what I did there)

Reply:
Added in https://github.com/sindresorhus/refined-github/pull/4252/commits/774a8798c8dc0232b60408e1d96f41992092c6f2